src/activity-window-port.js

```diff
@@ -224,14 +224,24 @@ export class ActivityWindowPort {
    */
   buildFeatures_() {
     const screen = this.win_.screen;
-    let w = Math.floor(Math.min(600, screen.width * 0.9));
-    let h = Math.floor(Math.min(600, screen.height * 0.9));
+    const availWidth = screen.availWidth || screen.width;
+    const availHeight = screen.availHeight || screen.height;
+    const maxWidth = Math.max(
+        this.win_.innerWidth * 0.5 || availWidth,
+        availWidth -
+        Math.max(0, (this.win_.outerWidth - this.win_.innerWidth) || 0));
+    const maxHeight = Math.max(
+        this.win_.innerHeight * 0.5 || availHeight,
+        availHeight -
```

Comment:
brief comment on how you arrived at this formula for maxHeight?

Reply:
Done